app.js

```diff
@@ -13,6 +13,28 @@ function hasFilter(name, filters) {
   return false;
 }
 
+function getParameterByName(name) {
+  var urlParams = window.location.search;
+  name = name.replace(/[\[\]]/g, "\\$&");
+
+  var results = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)").exec(urlParams);
+  
+  if (!results) return null;
+  if (!results[2]) return '';
+  return decodeURIComponent(results[2].replace(/\+/g, " "));
+}
+
+function parseTagsInUrl() {
```

Comment:
This function doesn't parse tags. It retrieves them form the URL using another function and then sets the corresponding inputs.
So I suggest changing its name to `setFiltersFromUrlParams`.